### ISSUE TITLE
Displayed metrics do not match real metrics in some cases

### ISSUE THREAD

**Issue by LexABzH** [OSS-Pole-Emploi/gabarit#127]
### Describe the bug

In some cases, the displayed metrics in a Keras model do not match real metrics.

## Concerned template

  - [x] NLP template
  - [x] NUM template
  - [x] VISION template
  - [ ] API template
  - [ ] How templates are generated - Jinja

### To Reproduce

Can't really provide a simple example.

![Capture](https://user-images.githubusercontent.com/20640502/212376647-820198d1-ac9e-4ad6-a1b6-b00b3c8d2f1a.PNG)

The previous example happened with a multilabel classification problem (2 labels) and with a highly unbalanced dataset (no dropout in the model).

### Expected behavior

Displayed metrics during training should match the real metrics.

Of course the training metrics can be different if one uses a dropout layer.

### Actual behavior

The current displayed metrics might not match the real metrics.

### Is this a regression?

That is, did this use to work the way you expected in the past?

- [ ] Yes
- [x] No

### Debug info

- Gabarit version: latest
- Python version: 3.8
- OS version: any
